plugin/PlayAssetDelivery/src/main/java/com/google/play/core/godot/assetpacks/PlayAssetDelivery.java

```diff
@@ -187,6 +187,29 @@ public Dictionary getPackLocations() {
     return PlayAssetDeliveryUtils.convertAssetPackLocationsToDictionary(packLocationsMap);
   }
 
+  /**
+   * Calls getPackStates(List<String> packNames) method in the Play Core Library. Requests download
+   * state or details for the specified asset packs. Emits getPackStatesSuccess and
+   * getPackStatesError signals when the underlying task succeeds/fails.
+   *
+   * @param packNames list of name for all the packs to request states
+   * @param signalID signalID used to track mapping of signals to Tasks
```

Comment:
nit: here and below, rather than repeating `signalID`: `@param signalID identifier used to track mapping of signals to Tasks`